src/test/java/org/magicdgs/thaplv/haplotypes/light/SNPpairTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2016, Daniel Gomez-Sanchez <daniel.gomez.sanchez@hotmail> All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification, are permitted
+ * provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this list of conditions
+ * and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions
+ * and the following disclaimer in the documentation and/or other materials provided with the
+ * distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse
+ * or promote products derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+ * THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+package org.magicdgs.thaplv.haplotypes.light;
+
+import static org.magicdgs.thaplv.haplotypes.light.LightGenotype.SNP;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @author Daniel Gómez-Sánchez
+ */
+@Deprecated
+public class SNPpairTest {
+
+    static SNPpair pair1_2, pair1_3, pair2_3, pair3_2;
+
+    @DataProvider(name = "pairsToTest")
+    public Object[][] setUpClass() throws Exception {
+        final LightGenotype variant1 = new LightGenotype("2L", 1,
+                new SNP[] {SNP.A, SNP.A, SNP.A, SNP.a, SNP.a, SNP.a, SNP.N});
+        final LightGenotype variant2 = new LightGenotype("2L", 2,
+                new SNP[] {SNP.a, SNP.a, SNP.A, SNP.A, SNP.a, SNP.N, SNP.N});
+        final LightGenotype variant3 = new LightGenotype("2L", 100,
+                new SNP[] {SNP.A, SNP.N, SNP.N, SNP.A, SNP.N, SNP.a, SNP.a});
+        pair1_2 = new SNPpair(variant1, variant2);
```

Comment:
\[FindBugs\] INFO: ST: Write to static field org\.magicdgs\.thaplv\.haplotypes\.light\.SNPpairTest\.pair1\_2 from instance method org\.magicdgs\.thaplv\.haplotypes\.light\.SNPpairTest\.setUpClass\(\)